lxd/config/map.go

```diff
@@ -182,6 +194,23 @@ func (m *Map) update(values map[string]string) ([]string, error) {
 // effectively revert it to the default. Return a boolean indicating whether
 // the value has changed, and error if something went wrong.
 func (m *Map) set(name string, value string, initial bool) (bool, error) {
+	// Bypass schema for user.* keys
+	if strings.HasPrefix(name, "user.") {
```

Comment:
We'll want to use `shared.IsUserConfig` once we have it, but other than that, looks good.